### PR TITLE
fixed background colors and layout for ourstory

### PR DESCRIPTION
**Pull request by RachellCalhoun**
I added the title for ourstory. I gave padding/margin etc for each post. Changed color to white.

### CODE REVIEW

**Comment by RachellCalhoun** [comment]
I think it's pretty much done. @hassanabidpk can you check?

**Comment by RachellCalhoun** [comment]
rachellourstory

File: content/models.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 class OurStory(models.Model):
     img = models.ImageField(blank=True, null=True)
     text = models.TextField()
-    color = models.CharField(max_length=6, blank=True)
+    color = models.CharField(max_length=20, blank=True)
```
Comment (hassanabidpk) [inline content/models.py:8]: Did you create migrations for it ?

File: content/templates/content/base.html [hunk 2]
```diff
@@ -22,10 +22,10 @@
         <link rel="stylesheet" type="text/css" href="{% static "css/content.css" %}">
       </head>
       <body>
-       <nav class="navbar  navbar-default">
+       <nav class="navbar  navbar-default navbar-fixed-top">
```
Comment (hassanabidpk) [inline content/templates/content/base.html:25]: :+1: about navbar-fixed-top

File: content/models.py (hunk 1 above)
Comment (RachellCalhoun) [inline content/models.py:8]: no, I guess i will? then push again?

Comment (hassanabidpk) [inline content/models.py:8]: Yup~ :+1: :pushpin:

**Pull request merged**